src/components/Header/DesktopNav/LogoLink.js

```diff
@@ -54,28 +52,15 @@ const LogoBehavior = ({ children, onMouseEnter, onMouseLeave }) => {
   );
 };
 
-const LogoLink = ({ squared, fillColorInitial, fillColorHover, textColor }) => {
-  const isHomePage = useContext(HomePageContext);
-
+const LogoLink = ({ fillColorInitial, fillColorHover, textColor }) => {
   const [fillColor, setFillColor] = useState(fillColorInitial);
   const onMouseEnter = () => fillColorHover && setFillColor(fillColorHover);
   const onMouseLeave = () => setFillColor(fillColorInitial);
 
   return (
     <LogoBehavior onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
-      {squared ? (
-        <SquaredLogo
-          css={iconStyles}
-          fillColor={fillColor}
-          textColor={textColor}
-        />
-      ) : isHomePage || fillColorInitial !== colors.text ? (
-        // Do not fall back to animated logo if there is a different color,
-        // since we cannot recolor that. Also force non-animated on home page.
-        <YldLogo css={iconStyles} fillColor={fillColor} />
-      ) : (
-        <img css={iconStyles} src={animatedLogo} alt="YLD animated logo" />
-      )}
+      (
```

Comment:
Leftover parentheses. Hopefully these are what causes the logo being slightly misaligned too